examples/src/main/scala/org/apache/spark/examples/streaming/KafkaWordCount.scala

```diff
@@ -65,6 +66,47 @@ object KafkaWordCount {
   }
 }
 
+/**
+  * Consumes messages from one or more topics in Kafka and does wordcount.
```

Comment:
Can you make this comment in more of the Spark style (e.g
/**
 * Consumes...
rather than
/**
  * Consumes...

See https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide for the multi-line-javadoc-style-comment suggestion.